typescript/commands/build.py

```diff
@@ -38,3 +40,4 @@ def compile_inferred_project(self, file_name, params=""):
             "cmd": cmd,
             "file_regex": "^(.+?)\\((\\d+),(\\d+)\\): (.+)$"
         })
+        self.build_parameters = params
```

Comment:
I mean if it is OK to pass ``None`` as the second argument to ``show_input_panel`` function. Also, does this change work on both sublime 2 and 3? We need to test for both